websites/B/Bungie/presence.ts

```diff
@@ -0,0 +1,303 @@
+
+var presence = new Presence({
+  clientId: "711393222252822539"
+});
+var browsingStamp = Math.floor(Date.now() / 1000);
+var title;
+var titleTwo;
+var titleThree;
+var titleFour;
+var titleFive;
+var titleSix;
+var titleSeven;
+var titleEight;
+var titleNine;
+var titleTen;
+var titleEleven;
+var titleTwelve;
+var titleThirteen;
+var titleFourteen;
+var titleFifteen;
+var titleSixteen;
+var titleSeventeen;
+var titleEighteen;
+var titleNinteen;
+var titleTwenty
+var titleTwentyOne;
+var titleTwentyTwo;
+var titleTwentyThree;
+var titleTwentyFour
```

Comment:
We suggest you use let or const instead of var

Reply:
tbh its not needed to have a different variable for every title, you could just use 1.

Reply:
Dude, the website had a bunch of things I needed to grab from it. Everything from the title of the forum post, to the title of the clan, to the title of the community post, to the author name (on different pages), the title of articles on different folder paths, and more... It's needed.

Reply:
Sorry, it's not.

Reply:
You can just keep setting the title to something different or even:
presenceData.details = "Viewing: " + document.querySelector("blabla").textContent
etc. It's not needed at all.

Reply:
Yeah it is as every title JavaScript path is different. Shoot, I can prove that.